resources/js/components/modules/templates/cardFlip.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+<div class="card-flip">
+  <section class="card-flip-description container">
+    <h1>{{slides.slide1.content.header}}</h1>
+    <p>{{slides.slide1.content.description}}</p>
+  </section>
+  <div class="card-flip-content">
```

Comment:
a div with a `.row` could live here and can you add a class of `.mb-5` to add a margin between the footer and the content